src/main/resources/application.yaml

```diff
@@ -75,7 +75,8 @@ idam:
   s2s-auth:
     url: ${S2S_URL:http://localhost:4502}
   s2s-authorised:
-    services: ${CPO_S2S_AUTHORISED_SERVICES:xui_webapp}
+    services: ${CPO_S2S_AUTHORISED_SERVICES:xui_webapp,payment_app}
+    read-only-services: ${CPO_S2S_READ_ONLY_SERVICES:xui_webapp}
```

Comment:
As in earlier comments, `payment_app` will be added to read only service once we know the name of the service with full CRUD permissions